packages/office-ui-fabric-react/src/components/Callout/examples/CalloutExample.scss

```diff
@@ -48,7 +48,7 @@
   }
 
   .ms-CalloutExample-inner {
-    height: 100%;
+    height: 200px;
```

Comment:
Are these example changes required for the callout to work properly? (Just making sure there are no backwards compatibility issues)